plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/HiveMetastoreModule.java

```diff
@@ -45,7 +45,9 @@ protected void setup(Binder binder)
             bindMetastoreModule("thrift", new ThriftMetastoreModule());
             bindMetastoreModule("file", new FileMetastoreModule());
             bindMetastoreModule("glue", new GlueMetastoreModule());
-            bindMetastoreModule("alluxio", new AlluxioMetastoreModule());
+            // Alluxio is not maintained by the Trino community and accumulates security vulnerabilities,
+            // so we want to avoid loading any classes for it if it is not used
```

Comment:
"
Load Alluxio metastore support through reflection. This makes Alluxio effectively an optional dependency
and allows deploying Trino without the Alluxio jar. Can be useful if the integration is unused and is flagged
but a security scanner.
"

Reply:
Updated 👍